From a2145190df8f4faa68a9662a2b5162dd87eca0a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Tue, 30 Aug 2016 18:25:21 +0200
Subject: [PATCH 34/39] MONITOR: Remove leftovers from kill_service
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Seems that wen I sent the v2 of ac35fe74 I attached the wrong pacth that
ended up being pushed.
The patch was incomplete as there are still some leftovers.

The .po and sssd-docs.pot were not touched as I do believe they are
autogenerated from Zanata.

Related:
https://fedorahosted.org/sssd/ticket/3052

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>
Reviewed-by: Petr Čech <pcech@redhat.com>
(cherry picked from commit 5b0735876aa66464b24cb7736a74fafd8ec82128)
---
 src/config/SSSDConfig/__init__.py.in |  1 -
 src/config/SSSDConfigTest.py         |  3 ---
 src/config/cfg_rules.ini             | 10 ----------
 src/config/etc/sssd.api.conf         |  2 --
 4 files changed, 16 deletions(-)

diff --git a/src/config/SSSDConfig/__init__.py.in b/src/config/SSSDConfig/__init__.py.in
index fb071270208341f4e993fef95af4e8dc6b58fdbd..0191920f93ab9016508e08785c25dd043c180c0b 100644
--- a/src/config/SSSDConfig/__init__.py.in
+++ b/src/config/SSSDConfig/__init__.py.in
@@ -46,7 +46,6 @@ option_strings = {
     'debug_microseconds' : _('Include microseconds in timestamps in debug logs'),
     'debug_to_files' : _('Write debug messages to logfiles'),
     'timeout' : _('Watchdog timeout before restarting service'),
-    'force_timeout' : _('Timeout between three failed ping checks and forcibly killing the service'),
     'command' : _('Command to start service'),
     'reconnection_retries' : _('Number of times to attempt connection to Data Providers'),
     'fd_limit' : _('The number of file descriptors that may be opened by this responder'),
diff --git a/src/config/SSSDConfigTest.py b/src/config/SSSDConfigTest.py
index 575a12450eec2e23e7fa30acf72030a0e7e07a50..6a0fdf0ea5215103b48dc8521a43ae945342c0e2 100755
--- a/src/config/SSSDConfigTest.py
+++ b/src/config/SSSDConfigTest.py
@@ -293,7 +293,6 @@ class SSSDConfigTestSSSDService(unittest.TestCase):
             'services',
             'domains',
             'timeout',
-            'force_timeout',
             'sbus_timeout',
             're_expression',
             'full_name_format',
@@ -505,7 +504,6 @@ class SSSDConfigTestSSSDDomain(unittest.TestCase):
             'min_id',
             'max_id',
             'timeout',
-            'force_timeout',
             'offline_timeout',
             'try_inotify',
             'command',
@@ -874,7 +872,6 @@ class SSSDConfigTestSSSDDomain(unittest.TestCase):
             'min_id',
             'max_id',
             'timeout',
-            'force_timeout',
             'offline_timeout',
             'try_inotify',
             'command',
diff --git a/src/config/cfg_rules.ini b/src/config/cfg_rules.ini
index a2c3fa2d5994dc051b72be17e13c512d7f124141..5e248066bd554d2a654a764f406f6b33c4d66733 100644
--- a/src/config/cfg_rules.ini
+++ b/src/config/cfg_rules.ini
@@ -23,7 +23,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # Monitor service
@@ -54,7 +53,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # Name service
@@ -92,7 +90,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # Authentication service
@@ -125,7 +122,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # sudo service
@@ -146,7 +142,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # autofs service
@@ -166,7 +161,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # ssh service
@@ -188,7 +182,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # PAC responder
@@ -209,7 +202,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 # InfoPipe responder
@@ -229,7 +221,6 @@ option = command
 option = reconnection_retries
 option = fd_limit
 option = client_idle_timeout
-option = force_timeout
 option = description
 
 #Available provider types
@@ -250,7 +241,6 @@ option = timeout
 option = try_inotify
 option = enumerate
 option = subdomain_enumerate
-option = force_timeout
 option = offline_timeout
 option = cache_credentials
 option = cache_credentials_minimal_first_factor_length
diff --git a/src/config/etc/sssd.api.conf b/src/config/etc/sssd.api.conf
index b2f20c5b71fa6bf94656d6270ccd08385a88c06e..525f939cd204f4d484caa7b490d85b0d50de00ef 100644
--- a/src/config/etc/sssd.api.conf
+++ b/src/config/etc/sssd.api.conf
@@ -13,7 +13,6 @@ command = str, None, false
 reconnection_retries = int, None, false
 fd_limit = int, None, false
 client_idle_timeout = int, None, false
-force_timeout = int, None, false
 description = str, None, false
 
 [sssd]
@@ -119,7 +118,6 @@ timeout = int, None, false
 try_inotify = bool, None, false
 enumerate = bool, None, false
 subdomain_enumerate = str, None, false
-force_timeout = int, None, false
 offline_timeout = int, None, false
 cache_credentials = bool, None, false
 cache_credentials_minimal_first_factor_length = int, None, false
-- 
2.9.3

